python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -60,12 +60,14 @@ def __init__(self, namePattern='', localDir='', **kwds):
             namePattern (str): is the pattern of the output file that has to be written into mass storage
             localDir (str): This is the optional local directory of a file to be uploaded to mass storage
         """
+        self._checkConfig()
```

Comment:
https://ganga-ci.cern.ch/jenkins/job/PR%20builder/2843/testReport/junit/python.Ganga.test.GPI.FileTests.TestSJIDName/TestSJIDName/test_a_jobSubmit/ is failing as there is no _safe_ default,
https://ganga-ci.cern.ch/jenkins/job/PR%20builder/2843/testReport/python.Ganga.test.GPI.FileTests.TestMassStorageClient/ passes because it's explicitly adding a path into the config by design (https://github.com/ganga-devs/ganga/blob/develop/python/Ganga/test/GPI/FileTests/TestMassStorageClient.py#L30)

Alternatively I suppose you could fix https://github.com/ganga-devs/ganga/blob/develop/python/Ganga/test/GPI/FileTests/TestSJIDName.py as it's the only test failing